.env

```
@@ -0,0 +1,3 @@
+STORYBOOK_DISABLE_TELEMETRY=true
+IN_STORYBOOK_SANDBOX=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IN_STORYBOOK_SANDBOX key should go before the STORYBOOK_DISABLE_TELEMETRY key

(UnorderedKey)